public/js/calendar.js

```diff
@@ -91,9 +83,7 @@ function generateMobileCalendar(data, calendarEl, room) {
       weekday: 'short',
       day: 'numeric'
     },
-    locale: 'hu',
     aspectRatio: 0.75,
-    selectable: true,
     events: data,
     select: info => {
```

Comment:
Change to `select: (info) => {` as above.

Reply:
We can talk about this too, personally I lean towards using parens in TS, since for type annotations we'll need them anyway.